src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/DraftAdjudicationService.kt

```diff
@@ -85,7 +84,7 @@ class DraftAdjudicationService(
       incidentDetails = IncidentDetails(
         locationId = locationId,
         dateTimeOfIncident = dateTimeOfIncident,
-        handoverDeadline = dateCalculationService.calculate48WorkingHoursFrom(dateTimeOfIncident)
+        handoverDeadline = dateTimeOfIncident.plusDays(2)
```

Comment:
I think it's worth centralising this logic - just have a hard-coded var or maybe a function

Reply:
sure.  did think this so will do